### ISSUE TITLE
Suggestion: ability to edit offline flights 

### ISSUE THREAD

**Issue by zartom** [ericberman/MyFlightbookAndroid#239]
After pack and go, same as can be done on iOS
![Screenshot_20210819-151526_MyFlightbook](https://user-images.githubusercontent.com/1769078/130461075-b00acb6b-10e2-44e3-bff3-96d827c25eeb.jpg)

**Comment by ericberman** [comment]
Ha, just went to start looking at this and at the relevant point in the code I found this comment:
            // Existing flights can't be saved for later.  No good reason for that except work.

**Comment by zartom** [comment]
Actually the functionality exists in iOS… this ticket is only to request the same for android

**Comment by ericberman** [comment]
Right, I know.  But iOS is a completely different code base, and the architecture of the iOS codebase there made the functionality come along for free.  Not so on the Android codebase.
